<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineView</class>
 <widget class="QWidget" name="MedicineView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedicineView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,7,3">
     <item>
      <layout class="QFormLayout" name="medicineOperationsFormLayout">
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item row="0" column="1">
        <widget class="QGroupBox" name="medicineOperationsGroupBox">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>230</height>
          </size>
         </property>
         <property name="title">
          <string>Prescription </string>
         </property>
         <widget class="QLabel" name="medicineNoLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Serial No</string>
          </property>
         </widget>
         <widget class="QLabel" name="medicineNameLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addMedicineButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QPushButton" name="removeMedicineButton">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>160</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="medicineNoLineEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="medicineNameLineEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QDateEdit" name="medicineStartDateEdit">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>90</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2025</year>
            <month>1</month>
            <day>6</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
         </widget>
         <widget class="QLabel" name="medicineStartDateLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Prod. Date</string>
          </property>
         </widget>
         <widget class="QLabel" name="medicineEndDateLabel">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>120</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Expr. Date</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="medicineEndDateEdit">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>120</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2025</year>
            <month>12</month>
            <day>12</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="medicinePieChartGroupBox">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>190</height>
          </size>
         </property>
         <property name="title">
          <string>Overview of Prescription Dates</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="pieVerticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="pieChartLabelHorizontalLayout">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="zone_patientNameLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(76, 175, 80);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="zone_patientSurnameLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Valid</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="patientFullNameHorizontalLayout">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="zone_patientNameLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 235, 59);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="zone_patientSurnameLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Expr. Date is Near</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="patientFullNameHorizontalLayout_2">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="zone_patientNameLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(244, 67, 54);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="zone_patientSurnameLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Out of Date</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="medicineListTableLayout">
       <item>
        <widget class="QGroupBox" name="medicineListGroupBox">
         <property name="minimumSize">
          <size>
           <width>530</width>
           <height>300</height>
          </size>
         </property>
         <property name="title">
          <string>Prescription List</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="medicineListVerticalLayout">
            <item>
             <widget class="QLineEdit" name="medicineSearchLineEdit">
              <property name="placeholderText">
               <string>Start typing for searching by name...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="medicinesTableView">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="patientsHorizontalLayout">
       <item>
        <widget class="QGroupBox" name="patientsGroupBox">
         <property name="title">
          <string>Patients Using the Drug</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="patientsVerticalLayout">
            <item>
             <widget class="QTableView" name="patientsTableView">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
